packages/lit-dev-content/site/_data/videos.js

```diff
@@ -80,6 +80,54 @@ const loadVideoData = () =>
       youtubeId: 'ri9FEl_hRTc',
       date: 'Oct 10 2023',
     },
+    {
+      title: 'Lit Labs: Task (graduated)',
+      summary: `This video covers @lit-labs/task, a package under development in Lit Labs. Watch for an overview, demos, package status and more! Originally streamed as part of the Lit Labs Lowdown event on October 24, 2022.`,
+      youtubeId: 'niWKuGhyE0M',
+      date: 'Jan 19 2023',
+    },
+    {
+      title: 'Lit Labs: Context (graduated)',
+      summary: `In this video, we cover @lit-labs/context, a package under development in Lit Labs. Watch for an overview, an explanation of the underlying Web Components Context Protocol, use cases, demos, package status and more! Originally streamed as part of the Lit Labs Lowdown event on October 24, 2022.`,
```

Comment:
What do you think about shortening these summaries. They're a bit long on the preview.